menu/navigation.json

```
@@ -3315,6 +3315,10 @@
                     "label": "Generate API keys for API and SMTP sending with IAM",
                     "slug": "generate-api-keys-for-tem-with-iam"
                   },
+                  {
+                    "label": "Set up a BIMI record",
+                    "slug": "bimi"
```

```suggestion
                    "slug": "configure-bimi"
```
maybe add a verb in the slug like `configure` or `setup`